go/plugins/compat_oai/openai/README.md

```diff
@@ -7,6 +7,34 @@ This plugin provides a simple interface for using OpenAI's services.
 - Go installed on your system
 - An OpenAI API key
 
+## Usage
+
+Here's a simple example of how to use the OpenAI plugin:
+
+```go
+// import "github.com/firebase/genkit/go/plugins/compat_oai/openai"
+// Initialize the OpenAI plugin with your API key
+oai := openai.NewPlugin(apiKey)
+
+// Initialize Genkit with the OpenAI plugin
+g, err := genkit.Init(ctx,
+    genkit.WithDefaultModel("openai/gpt-4o-mini"),
+    genkit.WithPlugins(oai),
+)
+if err != nil {
+    // handle errors
+}
+
+config := &ai.GenerationCommonConfig{
+    // define optional config fields
+}
+
+resp, err = genkit.Generate(ctx, g,
+    ai.WithPromptText("Write a short sentence about artificial intelligence."),
+    ai.WithConfig(config),
+)
```

Comment:
@apascal07 usage with helper

Reply:
Other plugins read the env variable. Can have it set in the req options, read it from the env by default

Reply:
reading from env variable in addition to passed parameter